pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -82,6 +92,11 @@ type Registry struct {
 	// and we pull from "example.com/bar/myimage:latest", the image will
 	// effectively be pulled from "example.com/foo/bar/myimage:latest".
 	// If no Prefix is specified, it defaults to the specified location.
+	// Prefix can also be in the format: "*.example.com" for matching
+	// subdomains. The wildcard should only be in the beginning and should also
+	// not contain any namespaces or special characters: "/", "@" or ":".
+	// DEPRECATED in favor of FindRegistry / PullSourcesFromReference instead
```

Comment:
I don't think that "deprecated" is the right term in this context. Prefix is still being used but we want to discourage users from using it with custom logic outside of `FindRegistry` etc.

Reply:
ack